examples/js/loaders/GLTFLoader.js

```diff
@@ -2432,6 +2435,22 @@ THREE.GLTFLoader = ( function () {
 
 	};
 
+	GLTFParser.prototype.assignUniqueName = function ( object, originalName ) {
+
+		var name = originalName;
+
+		for ( var i = 1; this.nodeNamesUsed[ name ]; ++ i ) {
+
+			name = originalName + '_' + i;
+
+		}
+
+		this.nodeNamesUsed[ name ] = true;
+
+		object.name = name;
+
+	};
+
```

Comment:
Nit, and no strong preference. But what do you think of removing `object` argument and returning the unique name instead? The methods will have less dependency with `object`.

```javascript
GLTFParser.prototype.createUniqueName = function ( originalName ) {

    var name = originalName;

    for ( var i = 1; this.nodeNamesUsed[ name ]; ++ i ) {

        name = originalName + '_' + i;

    }

    this.nodeNamesUsed[ name ] = true;

    return name;

};

node.name = parser.createUniqueName( node.name );
```

Reply:
Either is fine with me, yes 👍